server.js

```diff
@@ -1,33 +1,59 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import express, { response } from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+import netflixData from './data/netflix-titles.json';
 
 // Defines the port the app will run on. Defaults to 8080, but can be 
 // overridden when starting the server. For example:
-//
+
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
+const ERROR = { error: 'Request not found'};
+
+//Middlewares to enable cors and json body parsing
+app.use(cors());
+app.use(bodyParser.json());
+
+
+// Routes. Request is incoming and Response is outgoing.
+app.get('/', (request, response) => {
```

Comment:
Super small thing, but since endpoint methods always take `request` and `response` arguments, we can shorten them and everyone else will still know what they are : `(req, res)`